feature-libs/quote/core/connectors/quote.adapter.ts

```diff
@@ -5,52 +5,55 @@
  */
 
 import { PaginationModel } from '@spartacus/core';
-import { Observable } from 'rxjs';
 import {
-  QuoteList,
-  QuoteStarter,
+  Comment,
   Quote,
-  QuoteMetadata,
   QuoteActionType,
-  Comment,
   QuoteDiscount,
+  QuoteList,
+  QuoteMetadata,
+  QuoteStarter,
 } from '@spartacus/quote/root';
+import { Observable } from 'rxjs';
 
 export abstract class QuoteAdapter {
   /**
-   * Abstract method used to list of quotes for user
+   * Fetches a list of quotes
```

Comment:
Could we use `retrieves` for **getter** instead of `fetches / gets` ...?

Reply:
No, please see above